frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -309,7 +317,10 @@ export const EnvFromEditor = withDragDropContext(
                   variant="link"
                   isDisabled={addButtonDisabled}
                 >
-                  <PlusCircleIcon /> {addButtonLabel}
+                  <PlusCircleIcon />{' '}
+                  {addButtonLabel
+                    ? addButtonLabel
+                    : t('environment~Add all from Config Map or Secret')}
```

Comment:
Why do we need defaulting now when we didn't before?

Reply:
We had defaulting on line 348. I wasn't sure how to i18n .defaultProps objects, so I went this route.